{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -109,3 +109,15 @@ INSTANCE_THEME_FILE = './scss/{{ cookiecutter.project_shortname }}/theme.scss'
 
 # Instance's search theme entrypoint file. Path under `assets` folder.
 INSTANCE_SEARCH_THEME_FILE = './scss/{{ cookiecutter.project_shortname }}/search/theme.scss'
+{%- if cookiecutter.file_storage == 'S3'%}
+
+# Invenio-Files-Rest
+# ==================
+FILES_REST_STORAGE_FACTORY='invenio_s3.s3fs_storage_factory'
+
+# Invenio-S3
+# ==========
+S3_ENDPOINT_URL='http://localhost:9000/'
```

Comment:
Not https right?

Reply:
Sadly nope, but is not for production so shouldn't be an issue.